src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -875,6 +875,7 @@ Object AsyncRPCOperation_sendmany::perform_joinsplit(
             {info.vjsout[0], info.vjsout[1]};
     boost::array<size_t, ZC_NUM_JS_INPUTS> inputMap;
     boost::array<size_t, ZC_NUM_JS_OUTPUTS> outputMap;
+    std::function<int(int)> emptyFunc;
```

Comment:
If desired, we could define this in `src/random.h`, so that users can just treat it as a black-box identity generator, and the actual details of the implementation are right next to those of the shuffle itself.